### DISCUSSION TITLE
Visualizing the solid boundary

### DISCUSSION THREAD

**Discussion by sinahidd3n** [UvaCsl/HemoCell#23] [General discussion]
Hello, 
Is it possible with any method to visualize the solid boundary (e.g. pipe in the pipeflow) in the paraview along with other cells and flow profile?

Thanks
Sina

**Comment by paulmelis** [comment]
If you have an STL file like pipe.stl you can load this into paraview as an extra object

**Comment by DinushikaKarandeniya** [comment]
Hi sinahidd3n,

You can use the "Threshold" filter on the fluid output and select for the boundary value, that should give you the tool to differentiate between fluid and bounceback nodes. For this, the  OUTPUT_BOUNDARY needs to be enabled in the "case.cpp" file for the fluid outputs.

You could also try to overlay the STL file with the output, but then it needs to be correctly scaled and rotated, for both of which there are no standard methods...
